File: IRMC/app/src/main/java/tn/kassebiesprit/marwen/Institute.java
package irmc.esprit.tn.irmcmenufinale;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Institute {



    @SerializedName("id_inst")
    @Expose
    private Integer idInst;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("mail")
    @Expose
    private Object mail;
    @SerializedName("sigle")
    @Expose
    private String sigle;
    @SerializedName("code_postale")
    @Expose
    private String codePostale;
    @SerializedName("address")
    @Expose
    private String address;
    @SerializedName("longitude")
    @Expose
    private Double longitude;
    @SerializedName("latitude")
    @Expose
    private Double latitude;
    @SerializedName("website")
    @Expose
    private String website;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("type_acces")
    @Expose
    private String typeAcces;
    @SerializedName("image")
    @Expose
    private String image;
    @SerializedName("description")
    @Expose
    private String description;

    public Integer getIdInst() {
        return idInst;
    }

    public void setIdInst(Integer idInst) {
        this.idInst = idInst;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Object getMail() {
        return mail;
    }

    public void setMail(Object mail) {
        this.mail = mail;
    }

    public String getSigle() {
        return sigle;
    }

    public void setSigle(String sigle) {
        this.sigle = sigle;
    }

    public String getCodePostale() {
        return codePostale;
    }

    public void setCodePostale(String codePostale) {
        this.codePostale = codePostale;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public Double getLongitude() {
        return longitude;
    }

    public void setLongitude(Double longitude) {
        this.longitude = longitude;
    }

    public Double getLatitude() {
        return latitude;
    }

    public void setLatitude(Double latitude) {
        this.latitude = latitude;
    }

    public String getWebsite() {
        return website;
    }

    public void setWebsite(String website) {
        this.website = website;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getTypeAcces() {
        return typeAcces;
    }

    public void setTypeAcces(String typeAcces) {
        this.typeAcces = typeAcces;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

}

File: IRMC/Domain/Entity/institute.cs
namespace Data
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
   

    [Table("dbirmc.institute")]
    public  class institute
    {
        [Key]
        public int id_inst { get; set; }

        [StringLength(255)]
        public string address { get; set; }

        [StringLength(255)]
        public string description { get; set; }

        public float latitude { get; set; }

        public float longitude { get; set; }

        [StringLength(255)]
        public string name { get; set; }

        [StringLength(255)]
        public string type { get; set; }

        [StringLength(255)]
        public string code_postale { get; set; }

        [StringLength(255)]
        public string image { get; set; }

        [StringLength(255)]
        public string mail { get; set; }

        [StringLength(255)]
        public string sigle { get; set; }

        [StringLength(255)]
        public string type_acces { get; set; }

        [StringLength(255)]
        public string website { get; set; }
    }
}
